dolphinscheduler-api-test/dolphinscheduler-api-test-case/src/test/java/org/apache/dolphinscheduler/api.test/pages/project/WorkFlowDefinitionPage.java

```diff
@@ -0,0 +1,205 @@
+    private static String workFlowCode = null;
+    private static final Logger logger = LoggerFactory.getLogger(WorkFlowDefinitionPage.class);
+
+    public void getGenNumId(String sessionId, String projectName) {
+
+        Map<String, Object> params = new HashMap<>();
+        Map<String, String> headers = new HashMap<>();
+        params.put("genNum", 1);
+        headers.put(Constants.SESSION_ID_KEY, sessionId);
+        ProjectPage project = new ProjectPage();
+        String projectCode = project.getProjectCode(sessionId, projectName);
+        RequestClient requestClient = new RequestClient();
+        HttpResponse res = requestClient.get("/projects/"+projectCode+"/task-definition/gen-task-codes", headers, params);
+        ArrayList list = (ArrayList) res.body().data();
+        genNumId = list.get(0).toString();
+
+    }
+
+
+    public HttpResponse createWorkflow(String sessionId, String projectName, String workFlowName) {
+        Map<String, String> headers = new HashMap<>();
+        WorkFlowCreateRequestData workFlowCreateRequestData = new WorkFlowCreateRequestData();
+        TaskDefinitionRequestData taskDefinitionRequestData = new TaskDefinitionRequestData();
+        TaskRelationRequestData taskRelationRequestData = new TaskRelationRequestData();
+
+        TaskParamsMap taskParams = new TaskParamsMap();
+        ArrayList<Object> localParams = new ArrayList<>();
+        ArrayList<Object> resourceList = new ArrayList<>();
+        taskParams.localParams(localParams);
+        taskParams.resourceList(resourceList);
+        taskParams.rawScript("echo 123");
+
+        taskDefinitionRequestData.code(genNumId);
+        taskDefinitionRequestData.delayTime("0");
+        taskDefinitionRequestData.description("");
+        taskDefinitionRequestData.environmentCode("-1");
+        taskDefinitionRequestData.failRetryInterval("1");
+        taskDefinitionRequestData.failRetryTimes("0");
+        taskDefinitionRequestData.flag("YES");
+        taskDefinitionRequestData.name("echo_123");
+        taskDefinitionRequestData.taskParams(taskParams);
+        taskDefinitionRequestData.taskPriority("MEDIUM");
+        taskDefinitionRequestData.taskType("SHELL");
+        taskDefinitionRequestData.timeout(0);
+        taskDefinitionRequestData.timeoutFlag("CLOSE");
+        taskDefinitionRequestData.timeoutNotifyStrategy("");
+        taskDefinitionRequestData.workerGroup("default");
+
+        ArrayList<Object> taskDefinitionRequestDataList = new ArrayList<>();
+        taskDefinitionRequestDataList.add(taskDefinitionRequestData);
+
+        HashMap<String, Object> conditionParams = new HashMap();
+
+        taskRelationRequestData.name("");
+        taskRelationRequestData.preTaskCode(0);
+        taskRelationRequestData.preTaskVersion(0);
+        taskRelationRequestData.preTaskCode(0);
+        taskRelationRequestData.postTaskCode(genNumId);
+        taskRelationRequestData.conditionType("NONE");
```

Comment:
Same as L75.

Reply:
Thanks for the reminder, it has been fixed